tests_e2e/tests/scripts/agent_cgroups_process_check-unknown_process_check.py

```diff
@@ -81,9 +80,7 @@ def unknown_process_found(cpu_cgroup):
                 pid)), attempts=3)
         return found and retry_if_false(check_agent_quota_disabled, attempts=3)
 
-    cpu_cgroup, _ = get_unit_cgroup_paths(AGENT_SERVICE_NAME)
-
-    found: bool = retry_if_false(lambda: unknown_process_found(cpu_cgroup), attempts=3)
+    found: bool = retry_if_false(lambda: unknown_process_found(), attempts=3)       # pylint: disable=W0108
```

Comment:
can you check if lambda needed if function don't have args?

Reply:
the first arg to retry_if_false needs to be Callable. it won't accept bool as first arg

Reply:
i'll check if i can remove () and lambda

Reply:
removed